com.gitlab.j0chn.nextcloud_password_client.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.gitlab.j0chn.nextcloud_password_client",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "nextcloud_password_client",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.5.tar.gz",
+                    "sha256": "4338c6cab8af8dee6cdfd54e6218bd0533785f552c6162bb083f8dd28bf8fbbe"
+                }
+            ]
+        },
+
+        {
+            "name": "libsodium",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz",
+                    "sha256": "6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
```

Comment:
buildsystem = meson but a `CMAKE` option?

you get 
```
WARNING: Unknown options: "BUILD_SHARED_LIBS, CMAKE_BUILD_TYPE"
```

Reply:
on libsecret github page is written:
```
To build and install libsecret, you can use the following commands:

$ meson _build
$ ninja -C _build
$ ninja -C _build install
```
So cmake was no option for me.

Reply:
so why putting options for cmake? this is exactly my question.